test/integration/quic_http_integration_test.cc

```diff
@@ -704,41 +704,6 @@ TEST_P(QuicHttpIntegrationTest, CertVerificationFailure) {
   EXPECT_EQ(failure_reason, codec_client_->connection()->transportFailureReason());
 }
 
-// HTTP3 doesn't support 101 SwitchProtocol response code, the client should
-// reset the request.
-TEST_P(QuicHttpIntegrationTest, Reset101SwitchProtocolResponse) {
```

Comment:
if http/3 disallows why remove this?

Reply:
It's moved to protocol_integration_test

Reply:
Oh PEBKAC sorry, I saw the validation removal and the test "removal" and didn't look more carefully.